gamemodes/zombieplague/gamemode/sv_infectionmanager.lua

```diff
@@ -1,5 +1,6 @@
 InfectionManager = {}
 --Main function to infect a player, ZPInfectionEvent are called here
+	util.AddNetworkString("PlayerInfectedSomeone")
```

Comment:
I don't get it. We create this NetworkString, but we never call it. So how does the player gets notified?